change/@fluentui-contrib-react-keytips-2c77a34b-1b5c-4d63-a244-ad06854777c9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat(react-keytips): initial implementation",
```

the "scope" is covered by the change-file 


```suggestion
  "comment": "feat: initial implementation",
```